packages/admin-stories/src/admin/helpers/DataGenerators.ts

```diff
@@ -0,0 +1,34 @@
+export const generateRandomEmails = (amount: number, valueLength: number) => {
+    return Array.from(Array(amount).keys()).map(() => {
+        let string = "";
+        const chars = "abcdefghijklmnopqrstuvwxyz1234567890";
+        for (let i = 0; i < valueLength; i++) {
+            string += chars[Math.floor(Math.random() * chars.length)];
+        }
+        return `${string}@gmail.com`;
+    });
+};
```

Comment:
this could be replaced by faker.internet.email()

Reply:
haven't thought about that library :)